src/sql/src/session/vars.rs

```diff
@@ -1051,6 +1051,8 @@ pub enum NetworkPolicyError {
 pub struct SystemVars {
     /// Allows "unsafe" parameters to be set.
     allow_unsafe: bool,
+    /// max_credit_consumption_rate set by the license key.
+    max_credit_consumption_rate: Option<f64>,
```

Comment:
Instead of adding the license key's version of `max_credit_consumption_rate` to `SystemVars`, can we instead check the value from the `CatalogConfig` in the existing [`validate_resource_limit_numeric(...)`](https://github.com/ParkMyCar/materialize/blob/main/src/adapter/src/coord/ddl.rs#L1879-L1913) function?

I think this should prevent the need for drilling the license key value down so far and prevent the need for introducing a read_only concept of a system variable. wdyt?

Reply:
ah, yeah, this is much simpler. thanks!